{
    "type": "bundle",
    "id": "bundle--756b7d54-a40d-4764-b33c-b404cb8a6aae",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--586b9932-73a2-4138-aa95-97faa761c07e",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-02-14T00:19:43.731052Z",
            "modified": "2025-02-14T00:19:43.731052Z",
            "name": "CVE-2024-57782",
            "description": "An issue in Docker-proxy v18.09.0 allows attackers to cause a denial of service.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2024-57782"
                }
            ]
        }
    ]
}